node/service/src/chain_spec.rs

```diff
@@ -1820,13 +1823,18 @@ pub fn westend_local_testnet_config() -> Result<WestendChainSpec, String> {
 fn rococo_local_testnet_genesis(wasm_binary: &[u8]) -> rococo_runtime::GenesisConfig {
 	rococo_testnet_genesis(
 		wasm_binary,
-		vec![get_authority_keys_from_seed("Alice"), get_authority_keys_from_seed("Bob")],
+		vec![
+			get_authority_keys_from_seed("Alice"),
+			get_authority_keys_from_seed("Bob"),
+			get_authority_keys_from_seed("Charlie"),
```

Comment:
This will probably break tons of stuff, like tutorials and what else. Everything assumes 2 authorities. Could you please revert this.

You could just copy the function and have a different chain spec for your testing.

Reply:
Thank you for the catch! Do you think we could reuse wococo-local for that as we currently disable bridges modules in rococo runtime, so it doesn't serve its original purpose?

Reply:
I would have used a custom chain spec name (as you can use whatever you want), but as wococo isn't used currently it should be fine as well.